src/cli/cli.cpp

```diff
@@ -1228,6 +1229,39 @@ otError Interpreter::ProcessCounters(Arg aArgs[])
             ExitNow(error = OT_ERROR_INVALID_ARGS);
         }
     }
+    else if(aArgs[0]=="ip")
```

Comment:
Style/format issues (e.g., spacing)
```suggestion
    else if (aArgs[0] == "ip")
```



Suggest running `./script/make-pretty clang-format` to fix all formats. You can also check [coding-convention-and-style](https://github.com/openthread/openthread/blob/main/CONTRIBUTING.md#coding-conventions-and-style).